counter/package.json

```
@@ -1,29 +1,24 @@
 {
     "scripts": {
-        "start": "npx webpack-dev-server",
-        "build": "npx webpack"
+        "postinstall": "bower install bower.json && bower install",
```

I think this can be simplified (I had the more complicated one because I installed from two different `bower.json` files):

```suggestion
        "postinstall": "bower install",
```